### PR TITLE
Add Appveyor's Windows CI

### PR DESCRIPTION
**Pull request by timhaines**
TODO: 
- [ ] Get tests passing on windows

Closes https://github.com/percy/react-percy/issues/2

### CODE REVIEW

**Comment by thomashuston** [comment]
@timhaines Got this working, should be good to go!

File: packages/react-percy-webpack/src/entry/writeRenderEntry.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+import escapePathForWindows from './escapePathForWindows';
+import fs from 'fs';
 
 export default function writeRenderEntry(percyConfig, filePath, resolver = require.resolve) {
   fs.writeFileSync(
     filePath,
     `
     const url = require("url");
-    const renderer = require("${resolver(percyConfig.renderer)}");
+    const renderer = require("${escapePathForWindows(resolver(percyConfig.renderer))}");
```
Comment (timhaines) [inline packages/react-percy-webpack/src/entry/writeRenderEntry.js]: @thomashuston Is this going to substitute \ with / in any files that confusingly have \ in their name, even if not running on windows?

Comment (timhaines) [reply packages/react-percy-webpack/src/entry/writeRenderEntry.js]: While I wouldn't expect to see a file with \ in it's name, it's possible.  We had a guy with a : in a folder name a couple of weeks back.

Comment (timhaines) [reply packages/react-percy-webpack/src/entry/writeRenderEntry.js]: I think we pretty much use the same process as this in our other javascript packages though..

**Comment by timhaines** [comment]
Ha, GitHub doesn't let me approve this PR because I created it with the very first tiny step.  LGTM though!  🍀    I'm going to merge it.

**Pull request merged**